gulpfile.js

```diff
@@ -195,10 +204,7 @@ gulp.task('sprite', () => {
       })
     )
 
-  const imgStream = data.img
-    .pipe(changed('./src/assets/img/icons'))
-    .pipe(gulp.dest('./src/assets/img/icons'))
-
+  const imgStream = data.img.pipe(gulp.dest('./src/assets/img/icons'))
```

Comment:
Removal of `changed` due to an issue with Spritesmith not regenerating the files otherwise.